pkg/misconf/scanner.go

```diff
@@ -114,6 +115,7 @@ func newScanner(t detection.FileType, filePatterns []string, opt ScannerOption)
 	case detection.FileTypeAzureARM:
 		scanner = arm.New(opts...)
 	case detection.FileTypeCloudFormation:
+		opts = append(opts, cfscanner.WithParameterFiles(opt.CloudFormationParamVars...))
```

Comment:
The configuration files must be added to a separate FS using the `WithConfigFS` option. Or write to the current one via `configFiles = opt.CloudFormationParamVars`